src/remote.sh

```diff
@@ -28,6 +28,8 @@ function is_ssh_connection_configured()
   if [[ -z "$remote" && -z "$port" && -z "$user" ]]; then
     if [[ -n "${remote_file}" ]]; then
       ssh_cmd="ssh -q -o StrictHostKeyChecking=accept-new -o BatchMode=yes -o ConnectTimeout=5 -F ${remote_file} ${remote_file_host} exit"
+    else
+      return 2 # ENOENT
```

Comment:
Again, you implemented a test that expects this behavior before this behavior was implemented. As I said, you shouldn't introduce broken tests on a commit.